src/tiledb/cloud/bioimg/exportation.py

```diff
@@ -0,0 +1,105 @@
+from typing import Any, Dict, Mapping, Optional, Sequence, Tuple, Union
+
+import tiledb
+from tiledb.cloud._common.utils import logger
+from tiledb.cloud.bioimg.helpers import batch
+from tiledb.cloud.bioimg.helpers import get_uris
+from tiledb.cloud.bioimg.helpers import scale_calc
+
+DEFAULT_RESOURCES = {"cpu": "8", "memory": "4Gi"}
+DEFAULT_IMG_NAME = "3.9-imaging-dev"
+DEFAULT_DAG_NAME = "bioimg-exportation"
+
+
+def export(
+    source: Union[Sequence[str], str],
+    output: str,
+    config: Mapping[str, Any],
+    *args: Any,
+    taskgraph_name: Optional[str] = None,
+    num_batches: Optional[int] = None,
+    threads: Optional[int] = 8,
```

Comment:
nit: is `None` a valid value here? If not, this should just be `int`.

Reply:
I will remove `threads` arg it is a placeholder for now, we better add it when truly functional.